package.json

```diff
@@ -28,10 +28,10 @@
 		},
 		"./package.json": "./package.json"
 	},
-	"main": "dist/index.cjs",
-	"module": "dist/index.js",
-	"react-native": "dist/index.js",
-	"types": "dist/index.d.ts",
+	"main": "./dist/index.cjs",
+	"module": "./dist/index.js",
+	"react-native": "./dist/index.js",
+	"types": "./dist/index.d.ts",
```

Comment:
Why do we need `./` prefix now?

Reply:
The updated [`size-limit.cjs`](https://github.com/prismicio/prismic-client/blob/9fb5ae29ad7d0135b83389f254a17f7a3d4251f6/.size-limit.cjs) file reads `main`, `module`, and all `exports` (and perhaps it should read `react-native` as well). It dedupes the file list using a simple `Set`, which meant `dist/index.js` and `./dist/index.js` were treated as separate files.

I suppose we could add some `path.resolve()` logic

Reply:
The updated [`size-limit.cjs`](https://github.com/prismicio/prismic-client/blob/9fb5ae29ad7d0135b83389f254a17f7a3d4251f6/.size-limit.cjs) file reads `main`, `module`, and all `exports` (and perhaps it should read `react-native` as well). It dedupes the file list using a simple `Set`, which meant `dist/index.js` and `./dist/index.js` were treated as separate files.

I suppose we could add some `path.resolve()` logic to `size-limit.cjs` to make it smarter, but this solution was simpler.

Reply:
Makes sense to me! It's also more standard apparently so let's go